addons/web/static/src/core/errors/error_dialogs.js

```diff
@@ -17,62 +18,80 @@ odooExceptionTitleMap.set("odoo.exceptions.MissingError", _lt("Missing Record"))
 odooExceptionTitleMap.set("odoo.exceptions.UserError", _lt("User Error"));
 odooExceptionTitleMap.set("odoo.exceptions.ValidationError", _lt("Validation Error"));
 
+// -----------------------------------------------------------------------------
+// Generic Exception Dialog
+//
+// Define the props used on all Warning and Error dialogs.
+// -----------------------------------------------------------------------------
+class GenericExceptionDialog extends Dialog {}
+GenericExceptionDialog.props = Object.assign({}, Dialog.props, {
+    traceback: { type: String, optional: true },
+    message: { type: String, optional: true },
+    name: { type: String, optional: true },
+    exceptionName: { type: String, optional: true },
+    data: { type: Object, optional: true },
+    subType: { type: String, optional: true },
+    code: { type: Number, optional: true },
+    type: { type: String, optional: true },
+});
+
 // -----------------------------------------------------------------------------
 // Generic Error Dialog
 // -----------------------------------------------------------------------------
-export class ErrorDialog extends Component {
-    constructor() {
-        super(...arguments);
-        this.title = this.env._t("Odoo Error");
+export class ErrorDialog extends GenericExceptionDialog {
+    setup() {
+        super.setup();
+        this.props.title = this.env._t("Odoo Error");
```

Comment:
set this as a default props instead? I may be mistaken, but I think that if you try to instantiate an error dialog, and you do not specify a title (which seems reasonable as in this case, it is hardcoded by the component), you'll have a props validation error